pkg/plugins/common/k8s/names.go

```diff
@@ -15,8 +15,9 @@ const (
 	// The value has a format of a Kubernetes label name.
 	k8sNameComponent = "k8s.kuma.io/name"
 
-	// k8sSynced identifies that resource was synced
-	K8sSynced = "k8s.kuma.io/synced"
+	// K8sProcessed identifies that resource was already processed on creation. Specifically it's used to avoid
+	// creation of default resources twice when Kuma CP restarts
+	K8sProcessed = "k8s.kuma.io/processed"
```

Comment:
Since this is used in a really specific way, I'd suggest we use a specific name, like `kuma.io/mesh-defaults-generated`. 

Not clear to me whether `k8s.kuma.io` or just `kuma.io` is the right choice. The state being stored isn't kubernetes-specific, so maybe just `kuma.io`?

Reply:
I agree about `mesh-defaults-generated`, but I'm not sure about `kuma.io` vs `k8s.kuma.io`. All the names in this file have this `k8s.kuma.io` prefix. And I think it's Kubernetes specific because only `k8s.Store` uses it.

Reply:
¯\_(ツ)_/¯